vnext/template/cpp-app/src/App.h

```diff
@@ -2,12 +2,13 @@
 
 #include "App.xaml.g.h"
 
-{{#useWinUI3}}
+#include <CppWinRTIncludes.h>
```

Comment:
Same comment as in PCH. This is chunky. Would be best to include individual headers if possible.

Reply:
This would also leak namespace redefinition into the user project.